.github/CHANGELOG.md

```diff
@@ -229,6 +229,7 @@
 1. [MCUD] Add LSK for vert rev at destination - @tracernz (Mike)
 1. [MODEL] Add detailed rivet mesh to allow for ALDB+COMP+NORM ribbons - @bouveng (Johan Bouveng)
 1. [SD] Rewrite Bleed page to React - @RichardPilbery (tricky_dicky#3571)
+1. [BLEED] Converted bleed system to you mass flow instead of volume flow - @Gurgel100 (Pascal)
```

Comment:
"To you mass flow"
 ?

Reply:
Oh lol. Should say "To use mass flow". Fixed.